bsip-0083.md

```diff
@@ -0,0 +1,80 @@
+
+In short, this BSIP will put the BitAssets under the control of a new user account controlled by the stakeholders through a new type of voting. With this change, the newly appointed BitAssets manager will be able to independently maintain the parameters of their derivative asset without interference from the platform’s block producers and committee members. At the same time, the platform itself and its committee will not be unduly influenced by the demands of private product management. An advantage gained by this is that the BitAsset manager will be able to react appropriately to rapidly changing market conditions, as may be required to properly maintain a successful derivative asset.
+
+# Rationale
+Because the BitAssets are currently controlled by platform-wide governance mechanisms, and because these are not well suited to the management of smart coins, which may at times need to respond quickly to market conditions, the authors propose to reorganize these assets to utilize a dedicated stakeholder-voted multisignature authority and thereby enable these assets to update in synchronization with market ebbs and flows.
+
+The block producers will no longer provide feeds for BitAssets.
+
+# Specification
+The following technical changes are required:
+
+**Assignment of Accumulated Fees**
+
+At the hardfork date, the accumulated fees of the existing BitAssets will be transferred to the committee-account.
+
+**Creation of BITASSET.MANAGER asset**
+
+At the hardfork date, a new BITASSET.MANAGER asset will be created with the committee account as its owner and all flags and permissions disabled. The initial distribution will be made equal to the current BTS distribution. BTS that are locked up in open orders, in collateral positions, in HTLC objects or in vesting balances will be assigned to the owners of the respective positions. Other BTS positions are out of scope (e. g. stealth).
+Subsequently, tokens of this new asset can be freely transferred or traded.
+
+**Add a feeding_authority extension to BitAssets**
+
+An extension will be added to `asset_update_bitasset_operation` that allows specifying an elected authority (see BSIP-0084) for determining the list of feed producers. `asset_publish_feed_operation` will be adapted to check that authority if the extension is present on a BitAsset.
+
+**Addition of voting process through BITASSET.MANAGER**
+
+Two new elected authorities (see BSIP-0084) will be created based on the BITASSET.MANAGER asset with settings that match the current voting mechanism (regarding min/max_number and proxy_allowed). The first one is for controlling the new bitasset-manager account (see below), the second one for selecting price feeders.
+
+**Update to Committee-Controlled BitAsset Issuer Fields**
+
+At the hardfork date, a new bitasset-manager account will be created with its active authority set to the first of the new elected authorities, and with the owner authority controlled by the committee.
```

Comment:
Since people need time to vote, it would be a better idea to transfer the assets at a later date. IMHO we should not transfer the assets until there is a good-to-use UI to update votes.

Reply:
This BSIP is about *what* to do, not *when*.
Voting is largely controlled by proxies. Would it be helpful to create and pre-set proxies for these new elected authorities?

Reply:
Also note that both the new bitasset manager and the committee will have full control over the bitassets, i. e. if any problems arise the committee can set a fixed list of price feeders until voting kicks in.

Reply:
It does define when. "At the hardfork date" implies there is only one time defined for the hard fork.

Reply:
The important part about this proposal is that it clearly defines that bitasset management should be handed over from the committee to an independent asset manager. Allowing to postpone this for an indefinite amount of time is counterproductive.

Reply:
How about this:
1. We remove the automatic clearing and replacement of the price feeds from the BSIP.
2. We mandate that the committee switch to manual feeder selection for all committee-owner bitassets within two weeks after the BSIP has been approced.
3. We mandate that the committee (or bitasset manager) switch the price feed to the new elected authority within two weeks after the hardfork activation.

Reply:
I think we can use 2 or 3 timestamps for the protocol upgrade.

Reply:
IMO it is easier to let the committee activate the elected entities manually. It also gives us more flexibility if things don't quite work out as planned. And it is less work than implementing another hardfork.